### ISSUE TITLE
Enhancement: Display Comment in calendar entries

### ISSUE THREAD

**Issue by hrabbey** [dmedvinsky/mutt-ics#14]
Thanks much for mutt-ics.  I find it very useful.

A feature enhancement that would be helpful is if it could also display the first 10 lines of "COMMENT:", and append "..." if more than 10 lines.  

This would also help / be an alternate implementation of #5.

Comment is used as sort of the e-mail body in calendar replies / entries, and are hidden in the current version.  For the calendar entries from Outlook I have examined, it is a copy of of the plain text and HTML versions.

For viewing a long, large comment, it is easy enough for me, (and better in mutt in my opinion) to switch to the text or HTML version of the attachment via mutt's view-attachments menu.  However, with no indication of a comment existing, I have missed that there was more content available.

Example of current behavior and an example file is below.

```
hrabbey@localhost:~ $ pipsi upgrade mutt-ics                                                                                          
Requirement already up-to-date: mutt-ics in ./.local/venvs/mutt-ics/lib/python2.7/site-packages
Requirement already up-to-date: icalendar>=3.9.0 in ./.local/venvs/mutt-ics/lib/python2.7/site-packages (from mutt-ics)
Requirement already up-to-date: python-dateutil in ./.local/venvs/mutt-ics/lib/python2.7/site-packages (from icalendar>=3.9.0->mutt-ics)
Requirement already up-to-date: pytz in ./.local/venvs/mutt-ics/lib/python2.7/site-packages (from icalendar>=3.9.0->mutt-ics)
Requirement already up-to-date: setuptools in ./.local/venvs/mutt-ics/lib/python2.7/site-packages (from icalendar>=3.9.0->mutt-ics)
Cleaning up...
hrabbey@localhost:~ $ mutt-ics declined_with_comment.ics 
Subject:   Declined: Meeting Request
Organizer: hrAbbey@example.com
Start:     Thursday, 06 October 2016, 14:00 EDT
End:       Thursday, 06 October 2016, 15:00 EDT
Location:  Conference Room
Participants:
    pitiable.coworker@example.com
```

declined_with_comment.ics is
```
BEGIN:VCALENDAR
METHOD:REPLY
PRODID:Microsoft Exchange Server 2010
VERSION:2.0
BEGIN:VTIMEZONE
TZID:Eastern Standard Time
BEGIN:STANDARD
DTSTART:16010101T020000
TZOFFSETFROM:-0400
TZOFFSETTO:-0500
RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=1SU;BYMONTH=11
END:STANDARD
BEGIN:DAYLIGHT
DTSTART:16010101T020000
TZOFFSETFROM:-0500
TZOFFSETTO:-0400
RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=2SU;BYMONTH=3
END:DAYLIGHT
END:VTIMEZONE
BEGIN:VEVENT
ORGANIZER;CN=Howard Abbey:MAILTO:hrAbbey@example.com
ATTENDEE;PARTSTAT=DECLINED;CN=Coworker on Outlook:MAILTO:pitiable.coworker
 @example.com
COMMENT;LANGUAGE=en-US:Hello Howard\,\n\nThank you to make schedule at this
  time. Unfortunately\, I haven’t come to Michigan yet. Please tell other
  person\, or if possible\, please change the date.\n\nBest regards...
SUMMARY;LANGUAGE=en-US:Declined: Meeting Request
DTSTART;TZID=Eastern Standard Time:20161006T140000
DTEND;TZID=Eastern Standard Time:20161006T150000
UID:040000008200E00074C5B7101A82E0080000000050C13B98E61ED201000000000000000
 010000000CC45D2BE3422D34D9B21CBFABA861D35
CLASS:PUBLIC
PRIORITY:5
DTSTAMP:20161006T053443Z
TRANSP:OPAQUE  
STATUS:CONFIRMED
SEQUENCE:0
LOCATION;LANGUAGE=en-US:Conference Room
X-MICROSOFT-CDO-APPT-SEQUENCE:0
X-MICROSOFT-CDO-OWNERAPPTID:-400185376
X-MICROSOFT-CDO-BUSYSTATUS:BUSY
X-MICROSOFT-CDO-INTENDEDSTATUS:BUSY
X-MICROSOFT-CDO-ALLDAYEVENT:FALSE
X-MICROSOFT-CDO-IMPORTANCE:1
X-MICROSOFT-CDO-INSTTYPE:0
X-MICROSOFT-DISALLOW-COUNTER:FALSE
END:VEVENT
END:VCALENDAR
```

**Comment by dmedvinsky** [comment]
Hey Howard,

I don't really see the reason to only show first 10 lines of it. Are there the cases where someone quotes large blocks of emails in that reply or something?

Anyway, for now I added the display of the full content of the comment field. If I get convinced that it should be truncated, I will add it. It's quite easy to do.

**Comment by hrabbey** [comment]
Hello Dmitry,

Thank you for the update.  I tested it and it works well.

You are correct in my thoughts of avoiding large blocks of quoted e-mails.
However, that would easy enough to deal with, with a pipe through a pager.

It is probably more correct for the viewer to view the entire contents.

In addition, now that I have an easy way to view the comments, to my surprise I found most of them are already below the 10 lines anyway.  

So you made a good choice in not adding needless complexity.

Thanks much,
Howard
